common/components/ConfirmationModalTemplate/ConfirmationModalTemplate.tsx

```diff
@@ -42,8 +43,7 @@ export interface ConfirmButtonCBProps {
 }
 
 export interface OwnProps {
-  summary: React.ReactElement<any> | null;
-  details: React.ReactElement<any> | null;
+  isOpen?: boolean;
```

Comment:
Should accept a `body` parameter as per #940, which can either be this current implementation of `Body`, or a custom implementation